app/global/MainModule.java

```diff
@@ -260,6 +262,18 @@ private PeriodicMetrics providePeriodicMetrics(final MetricsFactory metricsFacto
         return periodicMetrics;
     }
 
+    @Provides
+    @com.google.inject.Singleton
+    @com.google.inject.name.Named("status-cache")
+    @SuppressFBWarnings("UPM_UNCALLED_PRIVATE_METHOD") // Invoked reflectively by Guice
+    private ActorRef provideStatusCache(
```

Comment:
Something isn't right here.  You're purporting to supply the status-cache actor, but actually providing the status actor.